cron/config/config.go

```diff
@@ -47,9 +47,12 @@ const (
 	metricExporter         string = "SCORECARD_METRIC_EXPORTER"
 	ciiDataBucketURL       string = "SCORECARD_CII_DATA_BUCKET_URL"
 	blacklistedChecks      string = "SCORECARD_BLACKLISTED_CHECKS"
-
+	// v2 results.
 	bigqueryTableV2       string = "SCORECARD_BIGQUERY_TABLEV2"
 	resultDataBucketURLV2 string = "SCORECARD_DATA_BUCKET_URLV2"
+	// v0 raw results.
+	rawBigqueryTableV0       string = "RAW_SCORECARD_BIGQUERY_TABLEV0"
+	rawResultDataBucketURLV0 string = "RAW_SCORECARD_DATA_BUCKET_URLV0"
```

Comment:
Remove the `v0` here and anywhere else. It unnecessarily adds a code cleanup task later.

Reply:
done